flake.nix

```diff
@@ -6,22 +6,39 @@
     foundry.url = "github:shazow/foundry.nix/monthly";
     flake-utils.url = "github:numtide/flake-utils";
     foundry.inputs.flake-utils.follows = "flake-utils";
+    nur.url = "github:nix-community/NUR";
+    goreleaser-nur.url = "github:goreleaser/nur";
   };
 
   outputs = inputs @ {
     self,
     nixpkgs,
     flake-utils,
     foundry,
+    nur,
+    goreleaser-nur,
     ...
   }:
     flake-utils.lib.eachDefaultSystem (system: let
-      pkgs = import nixpkgs {
-        inherit system;
-        overlays = [foundry.overlay];
-      };
+        isCrib = builtins.getEnv "IS_CRIB" == "true"; 
+        pkgs = import nixpkgs { inherit system; 
+          config = { allowUnfree = true; }; 
+          overlays = [
+            (final: prev: {
+              nur = import nur
+                {
+                  pkgs = prev;
+                  repoOverrides = {
+                    goreleaser = import goreleaser-nur { pkgs = prev; };
```

Comment:
This overrides NUR with the official goreleaser's nix NUR, since we want to get goreleaser pro directly from them.